examples/api/test/widgets/inherited_notifier/inherited_notifier.0_test.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/inherited_notifier/inherited_notifier.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('It rotate the spinners', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.InheritedNotifierExampleApp(),
+    );
+
+    final Iterable<Transform> widgets = tester.widgetList<Transform>(find.ancestor(of: find.text('Whee!'), matching: find.byType(Transform)));
```

Comment:
```suggestion
    final Iterable<Transform> widgets = tester.widgetList<Transform>(
      find.ancestor(of: find.text('Whee!'),
      matching: find.byType(Transform),
    ));
    
```
For readability, consider using multiline and adding an empty line.

Reply:
Sure, I updated it in [refactor: Use a utils matcher](https://github.com/flutter/flutter/pull/150344/commits/3c94fc58d1e330406584ec34bb8249a87062b9db)